doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -33,7 +33,19 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
-|no_changes_yet_note|
+the nRF70 Series
```

Comment:
I think we should clearly mention that customer should replace `CONFIG_WIFI_NRF700X*` Kconfig options with `CONFIG_WIFI_NRF700X*` Kconfig options. See below how clearly functions and Kconfig option is mentioned for lte_lc.

Reply:
Does it have to be an exhaustive list? They are too many :-(.

Reply:
discussed offline with @richabp we both felt it's better leave it to generic as there are too many Kconfigs and its a trivial namespace rename.

Reply:
Resolving this but if you disagree feel free to unresolve and we can discuss.

Reply:
I wasn't looking for a big list but a version with `*` character which I wrote in my original comment.
I'm trying to make migrations easier as we see complaints that migrations from one release to another are painful but we still continue to cause more and more changes during the migrations. This is a trivial namespace change but if we can save 5mins for one unexperienced developer, it's worth it (and I honestly believe we save a lot more no matter how trivial this feels).
But up to you and tech writers.

Reply:
Agreed, I have elaborated with an example, is that fine? https://github.com/nrfconnect/sdk-nrf/pull/16753/files#diff-73508775c4364c7221f61e1b098cd77eaf956e37eb209b707c649bc2b9a987c7.

Reply:
Looks good to me!